test/neuron/connection_test.exs

```diff
@@ -63,8 +66,8 @@ defmodule Neuron.ConnectionTest do
                  HTTPoison.post(
                    url,
                    query,
-                   "Content-Type": "application/graphql",
-                   "X-CUSTOM": "value"
+                   ["Content-Type": "application/graphql", "X-CUSTOM": "value"],
+                   []
```

Comment:
Could we add a new test here (similar to this one), checking that the`connection_opts` set are actually passed in when calling `HTTPoison.post/4`?

Reply:
makes sense. not sure why I didn't remember that in the first places

fixed

Reply:
That's perfect, thank you!